### ISSUE TITLE
Doc Encode + Sent Extract vs multi-layer LSTM

### ISSUE THREAD

**Issue by neverfox** [shashiongithub/Refresh#2]
Very interesting paper. Looking it over, there's something I cannot quite grasp about the architecture. What's the advantage of running the sentences through the Document Extractor LSTM, pulling out the state to initialize the Sentence Extractor LSTM and running the same sentences though that? Would you not get the same thing from running the sentences through a single multi-layer LSTM? Perhaps that's what you are doing in practice and the diagram is more there to aid interpretation, but I was nevertheless curious if that's true.

**Comment by shashiongithub** [comment]
We have separate document encoder and sentence extractor. The architecture is inspired from (Cheng and Lapata 2016). Idea is to use document encoder to learn a document representation first and then pass it to sentence extractor before start labelling sentences. We have not tried multi-layer LSTM in this paper. But it sounds an interesting idea.

**Comment by neverfox** [comment]
Right, but I guess I don't understand the theoretical advantages of doing this:

```
ignored   sent labels
   ^         ^
   |         |
DocExt -> SentExt ->  (hidden state, cell state)
   ^         ^
   |         |
 sents     sents
```
vs doing:
```
sent labels
   ^ 
   |
DocExt Layer 2 -> (hidden state, cell state)
   ^
   |
DocExt Layer 1 -> (hidden state, cell state)
   ^
   |
 sents
```

**Comment by shashiongithub** [comment]
We did not try the bottom architecture. (Cheng and Lapata 16) proposed a system where the outputs of DocExt are used. In our case, we started with (Cheng and Lapata 16). But it did not work very well, we further simplified it.
